### PR TITLE
Update team history and current members

### PR DESCRIPTION
**Pull request by ysims**
- Update the current team, including this year's RoboCup team and photo
- Moved last year's RoboCup team and photo to past members page
- Generalised some wording (xxxx-present rather than update the current year ever year)
- Added the summary of this RoboCup 

<https://deploy-preview-311--nubook.netlify.app/team/current-members>
<https://deploy-preview-311--nubook.netlify.app/team/history>

### CODE REVIEW

File: src/book/01-team/01-introduction/03-current-team.mdx [hunk 1]
```diff
@@ -30,45 +28,39 @@ Front (left to right): Jesse Perrin, Jason Disher, Joe Bailey, Ysobel Sims
...
+- **Jesse Williamson**: Undergraduate student studying Computer Science. Outreach Manager. DevOps and NUsight.
+
+- **Hengning "Henry" Xu**: Electrical Engineering PhD student. Control and reinforcement learning.
 
 ## Junior Members
 
-NUbots is currently running a recruitment drive! New team members will appear here soon.
+- **Ali Riyaz**, undergraduate Software Engineering
+- **Hayden Farley**, undergraduate combined Computer Systems Engineering and Computer Science
+- **Sebastian Catt**, undergraduate Computer Science
```
Comment (JesseWilliamson) [inline src/book/01-team/01-introduction/03-current-team.mdx:63]:
```suggestion
- **Ali Riyaz**, Undergraduate student studying Software Engineering
- **Hayden Farley**, Undergraduate student studying combined Computer Systems Engineering and Computer Science
- **Sebastian Catt**, Undergraduate student studying Computer Science
```

File: src/book/01-team/04-history/01-team-history.mdx [hunk 2]
```diff
@@ -14,10 +14,57 @@ authors:
...
-NUbots have competed in the Four Legged League (2002-2007), the Standard Platform League (2008-2011), the Humanoid KidSize League (2012-2017, 2021-2023) and the Humanoid TeenSize League (2018-2019).
+NUbots have competed in the Four Legged League (2002-2007), the Standard Platform League (2008-2011), the Humanoid KidSize League (2012-2017, 2021-present) and the Humanoid TeenSize League (2018-2019).
 
 NUbots became the Four Legged League world champions in 2006 in Germany. NUbots, in collaboration with the National University of Maynooth from Ireland, won the first Standard Platform League in 2008, as team NUManoid in Suzhou, China.
 
+## 2024: Eindhoven, The Netherlands
+
+NUbots returned to the Netherlands with eight student team members plus NUbots alumni and 4Ai Systems liaison Trent Houliston, who was reelected to the Humanoid League Technical Committee for a second year. Three team members attended their first RoboCup.
+
+NUbots achieved 4th place, with three games won and eight goals scored.
```
Comment (JesseWilliamson) [inline src/book/01-team/04-history/01-team-history.mdx:25]:
```suggestion
NUbots achieved 4th place with three games won and eight goals scored.
```

**Review by yippyy — DISMISSED** [review]

File: src/book/01-team/04-history/01-team-history.mdx [hunk 3]
```diff
@@ -14,10 +14,57 @@ authors:
...
+
+NUbots returned to the Netherlands with eight student team members plus NUbots alumni and 4Ai Systems liaison Trent Houliston, who was reelected to the Humanoid League Technical Committee for a second year. Three team members attended their first RoboCup.
+
+NUbots achieved 4th place with three games won and eight goals scored.
+
+**Major hardware changes**
+
+- Upgraded ankles and hip pitch and roll to X-Series servos.
+- Moved button panel to fixed back panel
+- New power solution
```
Comment (Claegtun) [inline src/book/01-team/04-history/01-team-history.mdx:31]: To what does this refer?

File: src/book/01-team/01-introduction/03-current-team.mdx [hunk 4]
```diff
@@ -30,45 +28,39 @@ Front (left to right): Jesse Perrin, Jason Disher, Joe Bailey, Ysobel Sims
 
 - **Joe Bailey**: Undergraduate student studying Computer Science. Vision data generation and benchmarking.
 
-- **Ethan Bird**: Undergraduate student studying Software Engineering. Localisation.
-
-- **Ronan Brattan**: Undergraduate student studying a combined degree in Computer Science and Mathematics. Demonstration development.
-
 - **Clayton Carlon**: Undergraduate student studying a combined degree in Electrical Engineering and Computer Systems Engineering. Electronics and audio signal processing.
```
Comment (Claegtun) [inline src/book/01-team/01-introduction/03-current-team.mdx:31]:
```suggestion
- **Clayton Carlon**: Bachelor of Computer-Systems Engineering (Honours), Bachelor of Electrical & Electronics Engineering (Honours). Electrical Engineering PhD candidate. Team Leader. Electronics and audio signal processing.
```

File: src/book/01-team/01-introduction/03-current-team.mdx [hunk 5]
```diff
@@ -30,45 +28,39 @@ Front (left to right): Jesse Perrin, Jason Disher, Joe Bailey, Ysobel Sims
...
-
-- **Liam Craft**: Bachelor of Computer Science. Studying Computing Honours. Laboratory Manager. Hardware design.
+- **Liam Craft**: Bachelor of Computer Science. Studying Computing Honours. Laboratory Manager. Hardware design and machine learning.
 
-- **Angelique Herfel**: Undergraduate student studying Software Engineering. Vision.
-
-- **Utkrisht Jain**: Undergraduate student studying Mechatronics Engineering. Hardware design and CAD.
+- **Angelique Herfel**: Undergraduate student studying Software Engineering. Vision and behaviour.
 
 - **Dexter Konijn**: Undergraduate student studying a combined degree in Computer Systems Engineering and Physics. Electronics.
```
Comment (Claegtun) [inline src/book/01-team/01-introduction/03-current-team.mdx:37]:
```suggestion
- **Dexter Konijn**: Undergraduate student studying a combined degree in Computer Systems Engineering and Physics. Deputy Team Leader. Electronics and localisation.
```

**Review by Claegtun — CHANGES_REQUESTED** [review]

**Review by Claegtun — APPROVED** [review]
Seems good to me.

**Pull request merged**